docs/tasks/argo.rst

```diff
@@ -0,0 +1,253 @@
+Modify the last line of the following yaml file as follows.
+
+.. code-block:: bash
+
+  kanctl create actionset --action backup --namespace kanister --blueprint [YOUR_BLUEPRINT_NAME] --statefulset [YOUR_STATEFULSET] --profile [YOUR_PROFILE_NAME] --secrets [YOUR_SECRETS_NAME]
+
+Then execute -
+
+.. code-block:: bash
+
+  cat<<EOF >> mysql-cron-wf.yaml
+  apiVersion: argoproj.io/v1alpha1
+  kind: CronWorkflow
+  metadata:
+    name: mysql-cron-wf
+  spec:
+    schedule: "*/5 * * * *"
+    concurrencyPolicy: "Replace"
+    workflowSpec:
+      entrypoint: automate-actionset
+      templates:
+        - name: automate-actionset
+          container:
+            image: ghcr.io/kanisterio/kanister-tools:v9.99.9-dev
+            command:
+              - /bin/bash
+              - -c
+              - |
+                microdnf install tar
+                curl -LO https://github.com/kanisterio/kanister/releases/download/0.80.0/kanister_0.80.0_linux_amd64.tar.gz
+                tar -C /usr/local/bin -xvf kanister_0.80.0_linux_amd64.tar.gz
+                kanctl create actionset --action backup --namespace kanister --blueprint mysql-blueprint --statefulset mysql-test/mysql-release --profile mysql-test/s3-profile-gd4kx --secrets mysql=mysql-test/mysql-release
+  EOF
+
+Note - Here, the cron-job is scheduled to run every 5 minutes. You may schedule
+it to run as per your requirements.
+
+Step 4 - Granting RBAC permissions.
+````````````````````````````````````````````
+
+Next, you will grant the Service Account in our ``argo`` namespace to access resources
+in the ``kanister`` and ``mysql-test`` namespace. This is required to create CRs based on
+the Secrets and StatefulSets that you provided in our previous step.
+You may read more about RBAC authorization here - RBAC_.
+
+1. Create a ClusterRole named ``cron-wf-manager`` having the permissions in
+   ``cluster-admin``.
+2. Grant these permissions to the default ServiceAccount named
+   ``default`` in the ``argo`` namespace.
+3. This will be done by creating a RoleBinding in the ``kanister``
+   and ``mysql-test`` namespaces.
+
+Execute the following -
+
+.. code-block:: bash
+
+  kubectl create rolebinding cron-wf-manager --clusterrole=cluster-admin --serviceaccount=argo:default -n kanister
+
+.. code-block:: bash
+
+  kubectl create rolebinding cron-wf-manager --clusterrole=cluster-admin --serviceaccount=argo:default -n mysql-test
+
+Step 5 - Launching the Cron Workflow
+````````````````````````````````````````````
+
+Lets launch the workflow in the ``argo`` namespace by running -
+
+.. code-block:: bash
+
+  argo cron create mysql-cron-wf.yaml -n argo
+
+Check if the workflow was created by running -
+
+.. code-block:: bash
+
+  argo cron list -n argo
+
+When the workflow runs, check if the ActionSet was created in the ``kanister`` namespace -
+
+.. code-block:: bash
+
+  kubectl get actionsets.cr.kanister.io -n kanister
+
+.. image:: img/argo-cron-created.png
```

Comment:
I think instead of having these commands' outputs as images, we should have them as code blocks here.
Also make sure that your machines' details are not in the output.

Reply:
Done.